packages/graphql-toolbox/src/contexts/utils.ts

```diff
@@ -168,6 +168,14 @@ export const resolveSelectedDatabaseName = (databases: Neo4jDatabase[]): string
         return usedDatabaseName;
     }
 
-    const defaultDatabase = databases?.find((database) => database.default) || undefined;
-    return defaultDatabase?.name || DEFAULT_DATABASE_NAME;
+    const defaultOrHomeDatabase = databases?.find((database) => database.default || database.home) || undefined;
```

Comment:
You could remove `|| undefined` if you want since `.find()` returns `YourThing | undefined`